### PR TITLE
Reviews

### PR DESCRIPTION
**Pull request by camilleion**
Created a new folder containing the document and code reviews rubric

### CODE REVIEW

File: documents/reviews/CodeReview.tex [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    Documentation (Including author): & \\
+    \hline
+    Structure/organization:
+    (does the work flow, are components grouped logically?) & \\
+    \hline
+    Syntax: & \\
+    \hline
+    Modulatity: & \\
+    \hline
+    Naming Conventions: & \\
```
Comment (braunsonm) [inline documents/reviews/CodeReview.tex]:
```suggestion
    Adheres to Coding Conventions: & \\
```
I think this should be broadened. Just following naming conventions is a small part of our code style

File: documents/reviews/CodeReview.tex [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	\hline
+    Concise:
+    (the purpose is achieved without undue complication) & \\
+	\hline
+    Documentation (Including author): & \\
+    \hline
+    Structure/organization:
+    (does the work flow, are components grouped logically?) & \\
+    \hline
+    Syntax: & \\
```
Comment (braunsonm) [inline documents/reviews/CodeReview.tex]: What does this mean? Syntax is compiler checked.

**Comment by braunsonm** [comment]
Everything else looks great Camille!

File: documents/reviews/CodeReview.tex (hunk 2 above)
Comment (camilleion) [reply documents/reviews/CodeReview.tex]: You're right, I'll take it out!

**Pull request merged**